flink-core/src/main/java/org/apache/flink/api/common/serialization/SimpleStringSchema.java

```diff
@@ -72,7 +72,7 @@ public Charset getCharset() {
 
 	@Override
 	public String deserialize(byte[] message) {
```

Comment:
I'd add `@Nullable` annotation.

Reply:
right

Reply:
I have to correct myself. It should be added to the method on the interface.